apps/v3-subgraph/subgraph.yaml

```diff
@@ -1,4 +1,4 @@
-specVersion: 0.0.4
+specVersion: 0.0.2
```

Comment:
Probably something I'm missing, but why drop the version?

Reply:
Good question, not sure why/how that is happening. Will check it out